templates/rustdoc/topbar.html

```diff
@@ -34,7 +39,7 @@
 
                     {%- if metadata.req_version|to_string == "latest" -%}
                     <li class="pure-menu-item">
-                        <a href="{{permalink_path()|safe}}" class="pure-menu-link description" id="permalink" title="Get a link to this specific version">
+                        <a href="{% if permalink_path is defined %}{{permalink_path|safe}}{% endif %}" class="pure-menu-link description" id="permalink" title="Get a link to this specific version">
```

Comment:
Shouldn't the if-block be outside if the `href` attribute? If `permalink_path` is undefined, then an empty `href=""` is rendered. I actually don't know what its semantics are.

Reply:
It's the goal actually. :wink: